db/migrate/20150610131245_change_start_and_end_time_fields_on_shift_to_datetime.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeStartAndEndTimeFieldsOnShiftToDatetime < ActiveRecord::Migration
+  def up
+    remove_column :shifts, :starting_time, :time
+    remove_column :shifts, :ending_time, :time
+    add_column :shifts, :starting_time, :datetime
+    add_column :shifts, :ending_time, :datetime
+  end
+
+  def down
+    remove_column :shifts, :starting_tame, :datetime
+    remove_column :shifts, :ending_time, :datetime
+    add_column :shifts, :starting_time, :time
+    add_column :shifts, :ending_time, :time
+  end
+end
```

Comment:
What's the logic behind turning `starting_time` and `ending_time` into a `datetime`? Surely a shift applies to every applicable date and so storing it as a time before made sense? (Also you have a typo on line 10.)